tests/php/Control/UserDefinedFormControllerTest.php

```diff
@@ -467,4 +468,31 @@ public function testRecipientAttachment()
         $nodata = $this->findEmail('nodata@example.com', 'no-reply@example.com', 'Email Subject');
         $this->assertEmpty($nodata['AttachedFiles'], 'Recipients with HideFormData do not receive attachment');
     }
+
+    public function testCreateNONExistedFolder()
```

Comment:
```suggestion
    public function testMissingFolderCreated()
```
Would the be a better name for what you're trying to test here?

Reply:
DONE